core/templates/dev/head/app.js

```diff
@@ -30,9 +30,10 @@ var oppia = angular.module(
     window.GLOBALS ? (window.GLOBALS.ADDITIONAL_ANGULAR_MODULES || [])
                    : []));
 
-// TODO(sll): Get this to read from a common JSON file; it's replicated in
-// feconf.
-oppia.constant('CATEGORY_LIST', GLOBALS.ALL_CATEGORIES);
+for (var constantName in CONSTANTS) {
+  oppia.constant(constantName, CONSTANTS[constantName]);
```

Comment:
Please add tests for this in ``appSpec.js``. Assuming the JS loading ``CONSTANTS`` works (you can setup a fake constants object in the spec), we can ensure that the constants are created correctly for each key in the constants object. If this code ever breaks, we really do not want to see a regression introduced because it could break constants across the entire frontend.